Form-Controls/styles.css

```diff
@@ -0,0 +1,66 @@
+* {
+    padding: 5px;
+    margin: 10px;
+    background-color: rgb(222, 206, 206);
+    font-family: monospace;
+  }
+  
+  header {
+    text-align: center;
+    font-size: 1.5rem;
+  }
+  
+  .para {
+    text-align: center;
+   
+  }
+  
+  .form {
+    display: flex;
+    justify-content: center;
+  }
+
+  label{
+    color: #0a0a09;
+    font-weight: bold;  
+  }
+  
+  input{
+    background-color: aliceblue;
+  }
+
+  .shirt-colours {
+    display: flex;
+    justify-content: center;
+    align-content: center;
+  }
+  
+  .shirt-size {
+    display: flex;
+    justify-content: center;
+    color: aliceblue;
+  }
+  
+  .delivery {
+    display: flex;
+    justify-content: center;
+  }
+  
+#size{
+    background-color: aliceblue;
+}
```

Comment:
typically we would avoid styling based on IDs in favor of classes. Why do we need to style the select background like this?

Reply:
i just change to class from ID. i thought  i should use id because its unique.